### PR TITLE
D8 nid 252

### PR DESCRIPTION
**Pull request by DuttonMa**
Update test install profile and README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,32 @@ Drupal source code for the NIDirect website: https://www.nidirect.gov.uk.
 
 ## Getting started
 
-This repository should be cloned as follows inside the /web directory of a working Drupal 8 NIDirect site:
+This repository should be cloned as follows inside the /web/profiles directory of a working Drupal 8 NIDirect site:
 
- git clone git@svegit01.thestables.net:dss/nidirect-d8-test-install-profile.git profiles
+ git clone git@svegit01.thestables.net:dss/nidirect-d8-test-install-profile.git custom
```
Comment (johangant) [inline README.md]: Change to git@github.com:dof-dss/nidirect-d8-test-install-profile.git ?

File: test_profile/config/sync/block.block.coldweatherpaymentcheckerblock.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+uuid: 9cbe0353-ac0b-47eb-963b-5b4bbb42ffaa
```
Comment (omahm) [inline test_profile/config/sync/block.block.coldweatherpaymentcheckerblock.yml]: Safe to remove this entire config. I had it placed to test the payments form.

File: README.md [hunk 3]
```diff
@@ -4,32 +4,32 @@ Drupal source code for the NIDirect website: https://www.nidirect.gov.uk.
...
+After doing this, edit profiles/custom/test_profile/config/sync/core.extension.yml and make sure that the install profile is set to 'test_profile' rather than 'standard'. Note that this may involve making two changes towards the bottom of the file, as shown in the following sample from the end of the core.extension.yml file:
 
   test_profile: 1000
   theme:
     stable: 0
     classy: 0
     bartik: 0
     seven: 0
-    stark: 0
+    stark: 0
```
Comment (johangant) [inline README.md:27]: See if your IDE can strip trailing whitespace on file save. That helps avoid this kind of thing which can help reduce noise on a diff

**Review by johangant — CHANGES_REQUESTED** [review]
Git URL to amend in the README but otherwise LGTM, thanks!

File: test_profile/config/sync/core.entity_form_display.node.external_link.default.yml [hunk 4]
```diff
@@ -110,4 +113,5 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
-hidden: {  }
+hidden:
+  field_top_level_theme: true
```
Comment (omahm) [inline test_profile/config/sync/core.entity_form_display.node.external_link.default.yml:117]: top level theme is removed further up and then set to hidden?

**Review by omahm — COMMENTED** [review]
Looks good, just a couple of comments

File: test_profile/config/sync/core.entity_form_display.node.external_link.default.yml (hunk 4 above)
Comment (DuttonMa) [reply test_profile/config/sync/core.entity_form_display.node.external_link.default.yml:117]: think it is just moved from 'visible' to 'hidden'

Comment (omahm) [reply test_profile/config/sync/core.entity_form_display.node.external_link.default.yml:117]: Is the theme taxo no longer a requirement?

**Pull request merged**